client/src/ui/molecules/submenu/index.tsx

```diff
@@ -58,6 +61,9 @@ export const Submenu = ({
                   className={`submenu-item ${
                     item.url.startsWith("https://") ? "external" : ""
                   }`}
+                  onClick={() =>
+                    gleanClick(`${MENU.CLICK_SUBMENU}: ${item.url}`)
```

Comment:
That's used in a lot of places, maybe we want to add something like `gleanEnabled`?

Reply:
Since we only measure clicks on submenu items with a url, this will effectively only measure clicks on the main menu and on the user menu. To distinguish these, I now added the menu id to the click source.